### DISCUSSION TITLE
Ldap authentication

### DISCUSSION THREAD

**Discussion by annemiekebuijs** [dunglas/symfony-docker#397] [Q&A]
It is amazing to see all this work you people do and i can just use it all for free.
Thank you so much.

I am working on 'dockarizing' all of my code.
Everything is going great, but for one thing, ldap authentication.
I want the enduser to authenticate against a existing, external active directory.
Without docker,  in symfony 6+, apache and php 8+ is it working just fine.
Now i have all of it in a dockerfile and now auth ldap is not working.
Can i get this to work with 'symfony docker'?

**Comment by annemiekebuijs** [comment]
I will try to get ldap authentication to work with caddy.

**Comment by maxhelias** [comment]
Hello, thank you for your message! :)

I have never used ldap authentication with symfony but the 'symfony docker' should have no impact if it is well configured.
_Note : Think that if your ldap server is in your docker-compose stack its host will be the service name_
